docs/index.html

```diff
@@ -45,10 +45,12 @@ <h1 id="in-a-nutshell">In a nutshell</h1>
 
         <p>SecHub represents a mechanism to integrate diverse security products like</p>
         <ul>
-          <li>static code analysis tools</li>
-          <li>web scanners</li>
-          <li>infrastructure scanners</li>
-          <li>… maybe more in future</li>
+          <li>SAST (static application security tools)</li>
```

Comment:
```suggestion
          <li>SAST (Static Application Security Testing)</li>
```

Reply:
ok, it's done

Reply:
@FanJups : Sorry, but please remove the gartner links ... It's not clear if the company allows deep links inside - I just added the links to give you the information.

But I think your idea about adding links for DAST and AST is very good.
https://en.wikipedia.org/wiki/Static_application_security_testing would be a good aceptable link for SAST